reactive/documentation/getting-started.adoc

```diff
@@ -0,0 +1,217 @@
+= Getting started with Hibernate Reactive
+:awestruct-layout: project-standard
+:awestruct-project: reactive
+:page-interpolate: true
+:toc:
+:toc-placement: preamble
+:hibernate-reactive-version: #{latest_release(page).version}
+:vertx-version: #{latest_release(page)['dependencies'].get_version('io.vertx', 'vertx-sql-client')}
+  
+[[reactive-gettingstarted]]
+
+Usage is very straightforward for anyone with any prior experience with
+Hibernate or JPA.
+
+== Example program
+
+There is a very simple example program in the link:https://github.com/hibernate/hibernate-reactive/tree/master/example[`example`
+directory on GitHub].
+
+== Including Hibernate Reactive in your project
+
+In your project, you need to add the dependency to Hibernate Reactive and
+the Vert.x clients to connect to the databases of your choice:
+
+[source]
+[subs="verbatim,attributes"]
+.Gradle dependencies
+----
+implementation org.hibernate.reactive:hibernate-reactive-core:{hibernate-reactive-version}
+runtimeOnly io.vertx:vertx-pg-client:{vertx-version}
+runtimeOnly io.vertx:vertx-mysql-client:{vertx-version}
+runtimeOnly io.vertx:vertx-db2-client:{vertx-version}
+----
```

Comment:
Shold we have a Maven example as well?

Reply:
The getting started now links to the documentation in Reactive. We can open an issue there.